pkgs/development/python-modules/stups-cli-support/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, clickclick
+, dnspython
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "stups-cli-support";
+  version = "1.1.20";
+  doCheck = false;
```

Comment:
we heavily prefer to add tests, this way can be sure a package still works, even when we bump dependencies. If the pypi package doesn't have it, then please retrieve from github using `fectchFromGitHub`.